conftest.py

```diff
@@ -18,6 +18,17 @@ def copy_fixtures(testdir):
     yield
 
 
+@pytest.fixture(autouse=True, scope='function')
+def cache_file_remove(cache_dir):
+    yield
+    _cache_file_path = os.path.join(cache_dir, 'port_target_cache')
+    if os.path.exists(_cache_file_path):
+        os.remove(_cache_file_path)
+        print(f'file in folder {_cache_file_path} was removed')
+    else:
+        print(f'file {_cache_file_path} doesn\'t exists')
```

Comment:
```suggestion
```

let's remove the lines which messed up the logs

Reply:
Updated